source/comply/audit-log.rst

```diff
@@ -1,369 +1,293 @@
-
-.. code-block:: none
-
-   "Levels": [
-      {"ID": 100, "Name": "audit-api"},
-      {"ID": 101, "Name": "audit-content"},
-      {"ID": 102, "Name": "audit-permissions"},
-      {"ID": 103, "Name": "audit-cli"},
-   ],
-
-where:
-
-- ``audit-api``: Enables output of REST API calls.
-- ``audit-content``: Enables output of API calls that generate content (e.g. ``create post``, ``create reaction``).
-- ``audit-permissions``: Enables output of all permissions failures.
-- ``audit-cli``: Enables output of legacy CLI calls.
-
-All levels can be viewed at ``mattermost-server/shared/mlog/levels.go``.
-
-.. Note::
-  - Logs are recorded asynchronously to reduce latency to the caller. 
-  - In version 5.x advanced logging supports hot-reloading of logger configuration. In versions 6.0 and greater, hot-reloading of configuration as been removed. 
-
-Configure audit log in Mattermost
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Configuring the Mattermost server to use the new audit log requires editing the ``config.json`` file directly. The audit settings output audit records to allow any combination of local file, syslog, and TCP socket targets.
-
-File target supports rotation and compression triggered by size and/or duration. Syslog target supports local and remote syslog servers, with or without TLS transport. TCP socket target can be configured with an IP address or domain name, port, and optional TLS certificate.
+Audit logging provides System Admins, including Security, IT/SRE, Compliance, and HR/PeopleOps teams, with a method of recording activities and events performed within a Mattermost workspace, such as access to the REST API or mmctl. Mattermost audit logging offers you control over where the audit logs are generated and stored, and applies a standard :doc:`JSON schema </comply/embedded-json-audit-log-schema>` to log output.
 
 .. note::
-   Mattermost v6.0 introduced a change to the logging engine library which changes how the logger is configured in Mattermost 5.x versions. Please refer to :download:`Advanced Logging sample file <../samples/sample-logger-config.json>` for a sample configuration file supported from v6.0. 
 
-   
-**Accessing configuration options for audit log**
+  - Logs are recorded asynchronously to reduce latency to the caller, and are stored separately from general logging.
+  - During short spans of inability to write to targets, the audit records buffer in memory with a configurable maximum size cap. Based on typical audit record volumes, it could take many minutes to fill the buffer. After that, the records are dropped, and the record drop event is logged.
+  - When using syslog remotely, we recommend writing to a local file so no records are lost. Records won’t be taken from the local file and sent back to syslog when syslog becomes available again.
```

Comment:
```suggestion
  - When using syslog remotely, we recommend writing to a local file so no records are lost. Records will be taken from the local file and sent to the remote syslog when syslog becomes available again.
```

Reply:
While I think this was the intent, I'm not sure where this functionality comes from.  Spooling to disk file and sending to remote server is not part of our implementation.

Reply:
> When using syslog remotely, we recommend writing to a local file so no records are lost. Records will be taken from the local file and sent to the remote syslog when syslog becomes available again.

@cwarnermm  Are we sure this section is correct?  I don't know what is providing this support for spooling to disk file and having it sent later.  Some third party software would be needed for that.

Reply:
This is pre-existing content that was brought forward. Given your input, I propose we remove this line.